timber/src/main/java/timber/log/Lumber.java

```diff
@@ -0,0 +1,121 @@
+package timber.log;
+
+import android.os.SystemClock;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+
+/** Timber timing logger utility */
+public final class Lumber {
+
+    public static class Woodchip {
+
+        private long mTime;
+        private String mLabel;
+
+        public long getTime() {
+            return mTime;
+        }
+
+        public String getLabel() {
+            return mLabel;
+        }
+
+        public Woodchip(long time, String label) {
+
+            mTime = time;
+            mLabel = label;
+        }
+    }
+
+    public static class Trace {
+
+        String mLabel = null;
+        List<Woodchip> mChips;
+
+        public Trace(String label) {
```

Comment:
@Avinash-Bhat maybe I don't get the convention properly, but I tried to follow it:

Timber -> Lumber
Tree -> Trace, Woodchip

What do you think?